.cirrus.yml

```diff
@@ -0,0 +1,95 @@
+env:
+  # Determine what we need here
+
+eks_container: &BUILD_TEMPLATE
+  dockerfile: .cirrus/Dockerfile
+  builder_subnet_id: ${CIRRUS_AWS_SUBNET}
+  builder_role: cirrus-builder
+  builder_image: docker-builder-v*
+  builder_instance_type: t2.small
+  cluster_name: ${CIRRUS_CLUSTER_NAME}
+  region: eu-central-1
+  namespace: default
+  docker_arguments:
+    CIRRUS_AWS_ACCOUNT: ${CIRRUS_AWS_ACCOUNT}
+
+yarn_cache_template: &YARN_CACHE_TEMPLATE
+  yarn_cache:
+    folders:
+      - '.yarn'
+    fingerprint_script: cat yarn.lock
+
+eslint_report_cache_template: &ESLINT_REPORT_CACHE_TEMPLATE
+  eslint_report_cache:
+    folder: 'build-reports/eslint-report'
+    fingerprint_script: echo $CIRRUS_BUILD_ID
+
+jest_coverage_cache_template: &JEST_COVERAGE_CACHE_TEMPLATE
+  jest_coverage_cache:
+    folder: 'build-reports/coverage'
```

Comment:
So apparently we're missing `/**` at the end of these paths: Ismael just came to my desk to let me know

Reply:
It's a non-issue here since we don't have nested folders. everything we cache lives directly inside the `build-reports/coverage` directory:

![Screenshot 2023-12-13 at 10 51 12](https://github.com/SonarSource/echoes-react/assets/97296331/8904a311-df43-4b09-926c-cc5843949b9b)

Of course, verify when we do the scan that there is **some** coverage
![Screenshot 2023-12-13 at 10 52 19](https://github.com/SonarSource/echoes-react/assets/97296331/9fd07662-7879-47e8-b7db-0ac33da7d1fe)

Reply:
also we can decide how to change reporting on CI, I didn't create a new `jest.config` for the ci to keep it simple

Reply:
as discussed, not an issue